geomstats/information_geometry/statistical_metric.py

```diff
@@ -0,0 +1,49 @@
+"""Statistical metric to equip a statistical manifold."""
+
+import geomstats.backend as gs
+from geomstats.geometry.riemannian_metric import RiemannianMetric
+
+
+class StatisticalMetric(RiemannianMetric):
+    """
+    Defines statistical metric and connection induced from a divergence.
+
+    Uses definitions provided in Nielson's An Elementary Introduction to
+    Information Geometry, Theorem 4 on page 15 (https://arxiv.org/abs/1808.08271)
```

Comment:
docstring missing parameter description (see other classes' docstring in geomstats)